gradoop-flink/src/main/java/org/gradoop/flink/model/api/operators/CallableGraph.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright © 2014 - 2019 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.flink.model.api.operators;
+
+import org.gradoop.flink.model.api.epgm.BaseGraph;
+import org.gradoop.flink.model.api.epgm.BaseGraphCollection;
+
+/**
+ * Defines callFor methods for graphs.
+ *
+ * @param <LG> graph type
+ * @param <GC> graph collection type
+ */
+public interface CallableGraph<LG extends BaseGraph, GC extends BaseGraphCollection> {
```

Comment:
I don't think the name of this interface is fitting, since the graph itself is not callable, but operators are callable with this graph as a parameter. Look at `java.util.concurrent.Callable` for example.
Maybe something like `BaseOperatorSupport`?

Reply:
At least Graph/GraphCollection has to be distinguishable. How about `BaseGraph(Collection)OperatorSupport`?

Reply:
Sounds good to me